src/Http/Authentication.Abstractions/src/IAuthenticationHandler.cs

```diff
@@ -18,19 +18,19 @@ public interface IAuthenticationHandler
     Task InitializeAsync(AuthenticationScheme scheme, HttpContext context);
 
     /// <summary>
-    /// Authenticate the current request.
+    /// Authenticate the request associated with the <see cref="HttpContext"/> passed to the InitializeAsync method.
```

Comment:
```suggestion
    /// Authenticate the current request using the <see cref="HttpContext"/> passed to the <see cref="InitializeAsync"/> method.
```

You should be able to use a `cref` for `InitializeAsync` too. I think we should still refer to "the current request". This should always ultimately be called by middleware handling an ongoing request. I think calling it "the current request" is fair in context.